NoSQL/Task1.md

```diff
@@ -0,0 +1,402 @@
+... |                            Delegating User Authentication and Product Subscription to a 3rd Party | 723f6f5f-3658-4449-90d0-4391d63e50a8
+
+(10 rows)
+```
+## 9. Use SELECT to COUNT ( the number Of imported rows. 
+It should match the number Of rows COPY reported as imported
+
+```
+cassandra@cqlsh:killrvideo> SELECT COUNT(*) FROM videos;
+
+ count
+-------
+   430
+
+(1 rows)
+
+Warnings :
+Aggregation query used without partition key
+```
+## 10. Use SELECT to find a row where the video_id=6c4cffb9-0dc4-1d59-af24-c960b5fc3652 
+
+```
+cassandra@cqlsh:killrvideo> SELECT * FROM videos where video_id = 6c4cffb9-0dc4-1d59-af24-c960b5fc3652;
+
+ video_id                             | added_date                      | description
+
+
+
+
+
+                                                                             | title
+                                       | user_id
+--------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------
+ 6c4cffb9-0dc4-1d59-af24-c960b5fc3652 | 2014-11-06 01:11:50.000000+0000 | Speaker: Luke Tillman, Language Evangelist at DataStaxnnKeyboard Cat, Nyan Cat, and of course the world famous Grumpy Cat--it seems like the Internet can’t get enough cat videos. If you were building an application to let users share and consume their fill of videos, how would you go about it? In this talk, we’ll take a look at the data model for KillrVideo, a sample video sharing application similar to YouTube where users can share videos, comment, rate them, and more. You’ll learn get a practical introduction to Cassandra data modeling, querying with CQL, how the application drives the data model, and how to shift your thinking from the relational world you probably have experience with. | Cassandra Day Denver 2014: A Cassandra Data Model for Serving up Cat Videos | fd3f7889-fc0c-43db-951c-7b77710898bc
+
+(1 rows)
+```
+
+Next we Will explore some Other 
+CQL commands that will come in handy, like TRUNCATE in a later exercise, we will show you how to add/remove (non-primary key) columns. 
+## 11. Let's remove the data from our table using TRUNCATE .truncate videos; 
+
+```
+cassandra@cqlsh:killrvideo> TRUNCATE TABLE videos;
```

Comment:
It is better to use schema name, not just table's. "killrvideo.videos", for safety.